dist/README.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: Testing
+nav_order: 1
+parent: Developer Overview
+---
+# RAPIDS Accelerator for Apache Spark Distribution Packaging
+
+Currently the distribution package is for making an uber jar with the Spark versions you need combined into a single jar.
+
+By default, it will only include the single version of Spark. This runs the profile `individual`.
+
+If you want to create a jar with multiple versions we currently have 4 options.
+
+1. Build for all Apache Spark versions and CDH with no SNAPSHOT versions of Spark, only released. Use `-Pallnosnapshots`.
```

Comment:
There's a lot of overlap here with what's in the contributing guide.  Seems like one of these should point to the other so we don't have to remember to update both locations when something changes.